pyigm/guis/igmguesses.py

```diff
@@ -1771,7 +1792,10 @@ def from_igmguesses_to_joebvp(infile, outfile):
     # Components
     comp_list = []
     for ii, key in enumerate(igmg_dict['cmps'].keys()):
-        comp = AbsComponent.from_dict(igmg_dict['cmps'][key], chk_sep=False, chk_data=False, chk_vel=True)
+        # QtCore.pyqtRemoveInputHook()
+        # pdb.set_trace()
+        # QtCore.pyqtRestoreInputHook()
+        comp = AbsComponent.from_dict(igmg_dict['cmps'][key], chk_sep=False, chk_data=False, chk_vel=False)
```

Comment:
For some reason this check is removing some of my lines. It may be an issue with backwards compatibility from LineLimits. It seems I have too many limits in some of my .json files that need to be clean at some point in the future. For this particular function, we can assume that the input .json file was generated by igmguesses, in which case all lines within a given components are good.